### PR TITLE
points encryption config observer to an encryption config file

### PR DESCRIPTION
**Pull request by p0lyn0mial**

### CODE REVIEW

File: pkg/operator/configobservation/configobservercontroller/observe_config_controller.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func NewConfigObserver(
 			project.ObserveProjectRequestMessage,
 			project.ObserveProjectRequestTemplateName,
 			proxy.NewProxyObserveFunc([]string{"workloadcontroller", "proxy"}),
-			observer.NewEncryptionConfigObserver(operatorclient.TargetNamespace, "/var/run/secrets/encryption-config"),
+			observer.NewEncryptionConfigObserver(operatorclient.TargetNamespace, "/var/run/secrets/encryption-config/encryption-config"),
```
Comment (p0lyn0mial) [inline pkg/operator/configobservation/configobservercontroller/observe_config_controller.go]: before it pointed to a directory

Comment (sttts) [reply pkg/operator/configobservation/configobservercontroller/observe_config_controller.go]: :+1:

**Pull request merged**